_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,155 @@
+
+- [How will the migration tool from a Bitcoin Core legacy wallet to a descriptor wallet work?]({{bse}}99624)
+  Michael Folkson outlines how a tool that migrates existing legacy wallets to
+  [descriptor][topic descriptors] wallets might work.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.2][C-Lightning 0.9.2] is the release for
+  the next maintenance version of C-Lightning.  It contains "new CLI-level
+  notifications, better channel state reporting, and stable plugin-hook
+  call ordering" in addition to other new features and bug fixes.  The
+  release notes do warn that [PSBTs][topic psbt] generated by this
+  version of C-Lightning will be rejected by some older versions of
+  Bitcoin Core (versions 0.20.0 and earlier); this is a consequence of a
+  change made to the [PSBT specification][news109 bips948] and [Bitcoin
+  Core][news105 core19215] to defend against a possible [fee overpayment
+  attack][].
+
+- [Bitcoin Core 0.21.0rc2][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+*Note: some commits to Bitcoin Core mentioned below apply to its
+development branch and so those changes will likely not be released
+until version 0.22, about six months after the release of the upcoming
+version 0.21.*
+
+- [Bitcoin Core #20305][] starts shifting RPC fee rate units to satoshis
+  per vbyte (sat/vB) rather than BTC per 1,000 vbytes (BTC/kvB) by introducing a
+  `fee_rate` parameter to the `sendtoaddress`,
+  `sendmany`, `fundrawtransaction`, and `walletcreatefundedpsbt` RPCs as well as
+  to the experimental [new][news116 send] `send` RPC. In addition, the `bumpfee` RPC `fee_rate`
+  option is changed from BTC/kB to sat/vB. Users are warned that the latter is a
+  breaking API change, but it should be relatively benign: the large (100,000
+  times) difference between BTC/kvB and sat/vB units means that a transaction
+  with a fee rate mistakenly calculated using BTC/kvB rather than sat/vB should
+  raise an error due to the fee rate being set too low. In the worst case, the
+  transaction may send at 1 sat/vB, but as [Replace-by-Fee][topic rbf] is active by
+  default when an explicit fee rate is used, the transaction fee can be
+  increased. The legacy `feeRate` option in RPCs `fundrawtransaction` and
```

Comment:
I think this is too much information for the newsletter. I suggest just saying that it's a breaking API change but should be benign. For more details see https://github.com/bitcoin/bitcoin/pull/20305#discussion_r520231413.

Reply:
Sure, I thought I should propose full disclosure of the changes that could affect users, but I'm too close to this to judge if it's too much detail for the newsletter and can definitely use the detailed version for the release note.